src/cli/src/error.rs

```diff
@@ -73,19 +70,17 @@ impl CliError {
     /// program should return when exiting after those errors.
     pub fn print(self) -> Result<()> {
         match &self {
-            Self::ConsumerError(ConsumerError::ClientError(FluvioError::ApiError(api))) => {
-                match api {
-                    ApiError::Code(ErrorCode::TopicAlreadyExists, _) => {
-                        println!("Topic already exists");
-                        Ok(())
-                    }
-                    ApiError::Code(ErrorCode::TopicNotFound, _) => {
-                        println!("Topic not found");
-                        Ok(())
-                    }
-                    _ => Err(self),
+            Self::ClientError(FluvioError::ApiError(api)) => match api {
```

Comment:
Ditto.  Let's keep ConsumerError

Reply:
I wonder if we can come up with a better name for it. Before we split the crates apart, there was no special type for ConsumerError, all of those variants simply lived in CliError, so this was returning to that previous pattern. If we're going to keep a new error type, it might make sense to come up with something that fits better.

Honestly, one thing I would really like to do is to only use `eyre::Result` in the entire CLI crate. That would make it so we would not need to keep track of the specific error types regardless of the libraries we use. This would essentially be equivalent to our current error-handling setup but without the hassle of maintaining and updating the CliError type. Let me know what you think.

Reply:
Is there reason to depend on `eye:Result`?